### ISSUE TITLE
Link text

### ISSUE THREAD

**Issue by oliverjam** [tacotoemeck/FaC-Calculator#2]
Here's an easy fix to make your page more accessible: interactive elements should always have a label describing what they do or where they go. Your footer links only contain icons, which means a screenreader user gets no description of where the link will take them when they focus it. There are a couple of ways to fix this:

1. Ensure there's a text label inside the `<a>` (you can [visually hide](https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html#hiding-content-visually) this if you think the icon is enough for sighted users)
1. Add an `aria-label` to the `<a>` with the descriptive text

**Comment by tacotoemeck** [comment]
That's awesome thanks for poiting it out. 

1. I've now added an inner text inside of the <a> tags with description of the links destination (i.e. "GitHub link"). Text is wrapped in <span> tags which has a class which visually hides the content but retains the description for the SR ( I've used an example from the article you suggested ). 

2. Secondly, I've added an aria-label to each link with a descriptive text.

**Comment by oliverjam** [comment]
Nice, thanks for changing. I wasn't clear enough in my suggestion: I meant you could do _either_ 1 or 2, not both 😅. If an element has an aria-label that overrides whatever text is inside, so there's not much point doing both

**Comment by tacotoemeck** [comment]
True, my mistake. Clearly, an accessibility overkill going with both options. 

Fixed now. 

Thanks!
